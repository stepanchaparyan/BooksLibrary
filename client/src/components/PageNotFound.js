import React, { Component } from 'react';

class PageNotFound extends Component {
  render() {
    return (
        <div className="pageNotFound">
            <h1>Page Not Found</h1>
        </div>
    );
  }
}

export { PageNotFound };