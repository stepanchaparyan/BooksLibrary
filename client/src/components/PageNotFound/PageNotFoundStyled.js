import styled from 'styled-components';

export const PageNotFoundContainer = styled.div`
  height: 50vh;
  display: flex;
  justify-content: center;
  align-items: center;
  color: red;
`;