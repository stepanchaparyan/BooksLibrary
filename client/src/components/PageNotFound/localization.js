import { defineMessages } from 'react-intl';

export default defineMessages({
  pageNotFound: {
    id: 'pageNotFound',
    defaultMessage: 'Page Not Found',
  }
});