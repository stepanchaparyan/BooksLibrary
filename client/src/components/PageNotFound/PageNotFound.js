import React, { Component } from 'react';
import { PageNotFoundContainer } from './PageNotFoundStyled';

class PageNotFound extends Component {
  render() {
    return (
        <PageNotFoundContainer>
            <h1>Page Not Found</h1>
        </PageNotFoundContainer>
    );
  }
}

export { PageNotFound };