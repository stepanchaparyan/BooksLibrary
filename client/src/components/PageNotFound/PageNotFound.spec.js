import React from "react";
import renderer from "react-test-renderer";
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import { PageNotFound } from "./PageNotFound.js";

function setupShallow() {
  const wrapper = shallow(<PageNotFound />);
  return { wrapper };
}

describe('First React component test with Enzyme', () => {
  it('renders without crashing', () => {
    const { wrapper } = setupShallow();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

