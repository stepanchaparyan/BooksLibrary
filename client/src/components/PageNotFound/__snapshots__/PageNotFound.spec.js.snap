// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<styled.div>
  <h1>
    Page Not Found
  </h1>
</styled.div>
`;
